content/en/docs/started/introduction.md

```diff
@@ -7,9 +7,10 @@ weight = 1
 ## What is Kubeflow
 
 Kubeflow is a community and ecosystem of open-source projects to address each stage in the
-machine learning (ML) lifecycle. It makes ML on Kubernetes simple, portable, and scalable.
-The goal of Kubeflow is to facilitate the orchestration of Kubernetes ML workloads and to empower
-users to deploy best-in-class open-source tools on any Cloud infrastructure.
+[machine learning (ML) lifecycle](/docs/started/architecture/#kubeflow-components-in-the-ml-lifecycle).
+It makes AI/ML on Kubernetes simple, portable, and scalable. We are an _ecosystem_ of Kubernetes
+based components for each stage in the ML Lifecycle with support for best-in-class
+open source tools and frameworks.
```

Comment:
I think we can remove the "We are..." sentence, because it effectively repeats the first sentence.